src/User.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Pokedex;
+
+
+class User
+{
+    private $email;
+
+    /**
+     * User constructor that checks if email address exists in the database.  Adds email address if it does not exist.
+     * @param AbstractEmail $email the validated email address from the user
+     * @param resource $db the database connection
+     */
+    public function __construct(AbstractEmail $email, \PDO $db)
+    {
+        $this->email = $email;
+
+        $stmt = $db->prepare('SELECT `id`, `email` FROM `users` WHERE `email` = :email;');
```

Comment:
no try catch, what happens if this query fails?

Reply:
Changed in 4df0906